.eslintignore

```diff
@@ -0,0 +1 @@
+vitest.config.ts
```

Comment:
nit: missing a newline at the end. most editors have a setting to prevent this, and it's specified in our .editorconfig file here if your editor supports that.